solr/CHANGES.txt

```diff
@@ -145,6 +145,8 @@ Bug Fixes
 
 * SOLR-16755: bin/solr's '-noprompt' option no longer works for examples (hossman, janhoy, Houston Putman)
 
+* SOLR-7609: Internal update requests should fail back to the client in some edge cases for shard splits. Use HTTP status 510 so the client can try the operation. (Alex Deparvu, David Smiley, Tomás Fernández Löbbe)
```

Comment:
```suggestion
* SOLR-7609: Internal update requests should fail back to the client in some edge cases for shard splits. Use HTTP status 510 so the client can retry the operation. (Alex Deparvu, David Smiley, Tomás Fernández Löbbe)
```

Reply:
thanks @tflobbe I pushed the change manually, GH wouldn't allow it for some reason